src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -151,7 +149,7 @@ public void AddOptionSet(OptionSet optionSet, DName optionalDisplayName = defaul
             AddEntity(optionSet, optionalDisplayName);
         }
 
-        internal bool TryGetSymbol(DName name, out IExternalEntity symbol, out DName displayName)

```

Comment:
>TryGetSymbol [](http://example.com/codeflow?start=22&length=12)

This is potentially internal-breaking; we should confirm with PAClient build.

Reply:
I did do a buddy build which ran successfully.